examples/nextjs-with-typescript/pages/MuxPlayer.tsx

```diff
@@ -48,27 +28,402 @@ const onEnded = console.log.bind(null, "ended");
 const onError = console.log.bind(null, "error");
 const onPlayerReady = console.log.bind(null, "playerready");
 
-function MuxPlayerPage() {
+const INITIAL_PRIMARY_COLOR = undefined;
+const INITIAL_SECONDARY_COLOR = undefined;
+const INITIAL_CONTROLS_BACKDROP_COLOR = undefined;
+const INITIAL_START_TIME = undefined;
+const INITIAL_THUMBNAIL_TIME = undefined;
+const INITIAL_DEBUG = undefined;
+const INITIAL_MUTED = undefined;
+const INITIAL_AUTOPLAY = undefined;
+const INITIAL_NOHOTKEYS = undefined;
+const INITIAL_DEFAULT_SHOW_REMAINING_TIME = undefined;
+const INITIAL_DEFAULT_HIDDEN_CAPTIONS = undefined;
+// const INITIAL_PLAYBACK_RATES = [0.25, 0.5, 1, 1.5, 2, 3];
+const INITIAL_PLAYBACK_RATES = undefined;
+const INITIAL_TITLE = undefined;
+// const INITIAL_ENV_KEY = "5e67cqdt7hgc9vkla7p0qch7q";
+const INITIAL_ENV_KEY = undefined;
+const INITIAL_SELECTED_CSS_VARS = {};
+const INITIAL_HOTKEYS = undefined;
+const INITIAL_FORWARD_SEEK_OFFSET = undefined;
+const INITIAL_BACKWARD_SEEK_OFFSET = undefined;
+const INITIAL_VOLUME = undefined;
+const INITIAL_LOOP = undefined;
+const INITIAL_CROSS_ORIGIN = undefined;
+const INITIAL_PLAYBACK_RATE = undefined;
```

Comment:
since these separate constants are not used anymore individually save some code lines and define them in the `DEFAULT_INITIAL_STATE` object?

Reply:
yup I'm down.